### ISSUE TITLE
Angels smelting extended

### ISSUE THREAD

**Issue by OsamaBinWOG** [Pezzawinkle/PezsMods#16]
With the latest updates to angels/madclowns I'm encountering the following error:

  52.144 Error ModManager.cpp:1574: Error in assignID: item with name 'angels-shielding-coil-copper' does not exist.
https://i.imgur.com/dcsy4wu.png

Only mods I'm running are Bobs+angels+madclowns. Not sure if you need any more info so ask if you need more.

**Comment by Dr-Jones** [comment]
I'm having the same issue. Here's the log from Factorio:

[factorio-previous.log](https://github.com/Pezzawinkle/PezsMods/files/6556195/factorio-previous.log)

This is without MadClowns.

**Comment by Pezzawinkle** [comment]
Ill take  look at this when I get home, I was sure it was working with the latest angels release.

**Comment by Pezzawinkle** [comment]
I am unable to replicate this error, can you please post a save file for me to check things with...

**Comment by Pezzawinkle** [comment]
Ok, this was one if statement not activating correctly due to a typo, ill get a release out in the next few mins

**Comment by OsamaBinWOG** [comment]
Thanks for the quick work!

**Comment by Dr-Jones** [comment]
Yup, confirmed working for me as well. Thanks for the quick turnaround!
